tests/scripts/all.sh

```diff
@@ -782,7 +782,7 @@ component_build_default_make_gcc_and_cxx () {
     make TEST_CPP=1
 }
 
-component_test_submodule_cmake () {
+component_test_use_psa_crypto_cmake () {
```

Comment:
This one actually *is* about submodule test and is not about `use_psa_crypto`. There are 4 `submodule` test and 1 `use_psa_crypto` test. The names should reflect that.

Reply:
Fat finger, just fixed that, thanks :)